v0/destinations/marketo/transform.js

```diff
@@ -194,7 +195,7 @@ const processIdentify = async (
   const attribute = constructPayload(traits, identifyConfig);
   Object.keys(leadTraitMapping).forEach(key => {
     const val = traits[key];
-    if (val) {
+    if (isDefinedAndNotNull(val)) {
```

Comment:
I think within here we need to handle the `null` values. Marketo uses `"NULL"` _as a string_ to clear out string fields.